src/modals/ImageModal/index.tsx

```diff
@@ -0,0 +1,33 @@
+import {createPortal} from 'react-dom';
+import {SFC} from 'types';
+import {useState} from 'react';
+import * as S from './Styles';
+import DefaultAvatar from '../../assets/default-avatar.svg';
```

Comment:
Can we also please sort/order the imports?
Please see other components how imports are sorted there.